config/checkstyle_input_suppressions.xml

```diff
@@ -2267,6 +2279,22 @@
             files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocUnclosedTagAndInvalidAtSeeReference.java"/>
   <suppress id="StyleValidationCommentInInputs"
             files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocWrongSingletonTagInJavadoc.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocInvalidAtSeeReference2.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocLeaveToken.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocNonTightHtmlTagsNoViolation.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocNonTightHtmlTagsVisitCount.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocParsingErrors2.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocTokensFail.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocTokensPass.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocTokensPass.java"/>
```

Comment:
Add `// violation` comment at lines where there is violation, and `// ok` for valid cases.